_includes/footer.html

```diff
@@ -1,18 +1,89 @@
-<footer class="site-footer" role="contentinfo">
+<div class="bg-base-light padding-y-1 line-height-sans-3 font-lang-4 text-ink">
   <div class="grid-container">
-    <div class="logo-links">
-      <a href="http://www.gsa.gov/">
-        <img src="{{ site.baseurl }}/assets/img/gsa-logo.svg" alt="GSA">
-      </a>
-    </div>
-    <div class="footer-text">
-      <p class="usa-text-small">A project of GSA’s <a href="https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services">Technology Transformation Services</a>, maintained by the <a href="https://accessibility.18f.gov/">Accessibility Guild</a>.</p>
-      <p class="usa-text-small">This website is hosted on <a href="https://federalist.18f.gov/">Federalist</a>.</p>
-    </div>
-    <div class="footer-buttons">
-      <a class="footer-button usa-text-small" href="https://github.com/18F/accessibility-playbook/edit/main/{{ page.path }}"><i class="fa fa-pencil" aria-hidden="true"></i>Edit this page</a>
-      <a class="footer-button usa-text-small" href="https://github.com/18F/accessibility-playbook/issues/"><i class="fa fa-github" aria-hidden="true"></i>File an issue</a>
-    </div>
+    <b>Contact:</b> Questions or comments? Please file a
+    <a href="https://github.com/18F/accessibility-playbook/issues/" class="text-ink text-bold hover:text-no-underline">GitHub issue</a>
+    or email
+    <a href="mailto:g-accessibility@gsa.gov" class="text-ink text-bold hover:text-no-underline">g-accessibility@gsa.gov</a>.
+  </div>
+</div>
+
+<footer class="site-footer" role="contentinfo">
```

Comment:
`role="contentinfo"` is redundant for a standalone `footer` element. It doesn't hurt anything, but it is extra code that only restates the footer's default/inherent role.

Reply:
this is now removed